optuna/samplers/grid.py

```diff
@@ -48,10 +52,9 @@ def objective(trial):
 
     Note:
 
-        :class:`~optuna.samplers.GridSampler` raises an error if all combinations in the passed
-        ``search_space`` has already been evaluated. Please make sure that unnecessary trials do
-        not run during optimization by properly setting ``n_trials`` in the
-        :func:`~optuna.study.Study.optimize` method.
+        :class:`~optuna.samplers.GridSampler` automatically stops the optimization if all
+        combinations in the passed ``search_space`` has already been evaluated, internally
```

Comment:
Thank you for your update!

[nit] I guess `has` is a typo for `have`.
```suggestion
        combinations in the passed ``search_space`` have already been evaluated, internally
```

Reply:
Thanks! I merged the change.